### ISSUE TITLE
Latest release broken?

### ISSUE THREAD

**Issue by uosis** [tlabs-data/tablesaw-parquet#76]
I was trying to do simple parquet write as per example (using Scala REPL for ease of illustration here):

```scala
> import $ivy.`tech.tablesaw:tablesaw-core:0.43.1`
> import $ivy.`net.tlabs-data:tablesaw_0.43.1-parquet:0.10.1`
> val table = tech.tablesaw.api.Table.create("t").addColumns(tech.tablesaw.api.DoubleColumn.create("c", 1, 2, 3))
> new net.tlabs.tablesaw.parquet.TablesawParquetWriter().write(table, net.tlabs.tablesaw.parquet.TablesawParquetWriteOptions.builder("test.parquet").build())
java.lang.NullPointerException: Cannot invoke "tech.tablesaw.api.DoubleColumn.getDouble(int)" because "this.doubleColumns[colIndex]" is null
  net.tlabs.tablesaw.parquet.TableProxy.getDouble(TableProxy.java:225)
```

Offending code seems to be [here](https://github.com/tlabs-data/tablesaw-parquet/blob/master/src/main/java/net/tlabs/tablesaw/parquet/TableProxy.java#L104) - that comparison will never be true because `ColumnType` singleton init is broken and will return different instances. I filed an [issue](https://github.com/jtablesaw/tablesaw/issues/1205) with tablesaw, but was wondering if there is some workaround? This seems like a very basic use case that tests would have caught?

**Comment by ccleva** [comment]
Hi @uosis, thanks for your feedback. I was able to reproduce the problem in Java, so I will implement a fix for this and try to come up with a workaround in the meantime.

**Comment by ccleva** [comment]
I confirm using the equal method instead of using reference equality prevents this issue. This will be fixed in the next release.

In the meantime, a call to `tech.tablesaw.api.ColumnType.values()` (or another static method) before creating the table ensures the references will be the same.

I am as surprised as you this was not caught by existing tests.

**Comment by uosis** [comment]
> I confirm using the equal method instead of using reference equality prevents this issue. This will be fixed in the next release.

Great, thank you for a prompt response!

> In the meantime, a call to tech.tablesaw.api.ColumnType.values() (or another static method) before creating the table ensures the references will be the same.

Yeah, that makes the example work, but it causes other init problems as outlined in tablesaw [issue](https://github.com/jtablesaw/tablesaw/issues/1205). It's probably the best we can do at this point however.

**Comment by ccleva** [comment]
Hi @uosis, this is now fixed in [v0.11.0](https://github.com/tlabs-data/tablesaw-parquet/releases/tag/0.43.1-v0.11.0).

Thanks again for reporting the issue and pointing to the solution.

**Comment by uosis** [comment]
> Hi @uosis, this is now fixed in [v0.11.0](https://github.com/tlabs-data/tablesaw-parquet/releases/tag/0.43.1-v0.11.0).
> 
> Thanks again for reporting the issue and pointing to the solution.

Awesome, thanks for fixing it so quickly!
